package-scripts.js

```diff
@@ -6,13 +6,10 @@ module.exports = {
     },
     test: {
       default: {
-        description: 'Runs AVA with nyc (which is configured in package.json)',
-        script: 'cross-env NODE_ENV=test nyc ava',
-      },
-      watch: {
-        description: 'Run AVA in watch mode',
-        script: 'ava -w --require babel-register',
+        description: `This runs jest with coverage. If we're on Travis, then we'll ignore the cache (just in case).`,
+        script: `jest --coverage ${process.env.CI ? '--no-cache' : ''}`,
```

Comment:
btw. if there is no cache (and on the CI most of the time there won't be), you don't really need to explicitly add --no-cache. It doesn't hurt either, I guess.